spec/rubocop/cop/salsify/rails_application_mailer_spec.rb

```diff
@@ -1,76 +1,75 @@
 # frozen_string_literal: true
 
 describe RuboCop::Cop::Salsify::RailsApplicationMailer do
-  let(:msgs) { [described_class::MSG] }
-  let(:highlights) { ['ActionMailer::Base'] }
+  let(:highlights) { 'ActionMailer::Base' }
 
   subject(:cop) { described_class.new }
 
   it "allows ApplicationMailer to be defined" do
     source = "class ApplicationMailer < ActionMailer::Base\nend"
-    inspect_source(source)
-    expect(cop.offenses).to be_empty
+    inspection = inspect_source(source)
```

Comment:
API change here. These two cops should probably be refactored to use the test helpers the other cops are using.

Reply:
I'm happy to make any adjustments people suggest, I just have very little context on general Rails development at Salsify to know which of these are good to keep. I'll re-enable this one.

(Also I'm assuming that you meant this comment on the MailerNaming cop in rubocop-rails, but it's showing up nested under some of my test changes.)

Reply:
I meant that we don't even need to check for mailer naming, since there's no reason for us to use `ApplicationMailer` or the legacy name `ActionMailer::Base`

Therefore we could delete the custom cop altogether,
https://github.com/salsify/salsify_rubocop/blob/master/lib/rubocop/cop/salsify/rails_application_mailer.rb
and this corresponding spec, as opposed to refactoring to use the new helpers.

But I wouldn't make that a blocker for this PR.

Reply:
The `MailerNaming` cop is slightly different, it's about having a `Mailer` suffix on subclass names.
https://docs.rubocop.org/rubocop-rails/cops_rails.html#railsmailername

So hypothetically you could restore the `MailerName` lines you deleted in https://github.com/salsify/salsify_rubocop/pull/47/commits/e5fed3a73c8f110d787ff67647f1ddd7ce4c9999 since it should never arise in practice, for the same reason that we don't use Rails mailers.